src/main/menu/templates/view.js

```diff
@@ -104,7 +104,7 @@ export default function (keybindings) {
       }
     })
     viewMenu.submenu.push({
-      label: 'Reload',
+      label: 'Reload (for Development)',
```

Comment:
I would prefer `Reload window` because it's only enabled in development/debug mode.